data/archipelago/scripts/patches/extend_forge_item_convert.lua

```diff
@@ -0,0 +1,56 @@
+local function ap_extend_forge_item_convert()
+	local entity_id = GetUpdatedEntityID()
+	local pos_x, pos_y = EntityGetTransform(entity_id)
+	local converted = false
+	for _, id in pairs(EntityGetInRadiusWithTag(pos_x, pos_y, 70, "tablet")) do
+		if EntityGetRootEntity(id) == id then
+			local x, y = EntityGetTransform(id)
+			local item_comps = EntityGetComponent( id, "ItemComponent" )
```

Comment:
```suggestion
			local item_comps = EntityGetComponent( id, "ItemComponent" ) or {}
```

Then you can get rid of the if statement that checks it.